api-server/src/roster/models.py

```diff
@@ -61,6 +63,7 @@ class DCSModules(models.Model):
                                    default=module_types[0])
 
     service = models.CharField(choices=[(x, x) for x in services], blank=True, null=True, max_length=64)
+    dcs_livery_label = models.CharField(max_length=128)
```

Comment:
is this something that needs to be stored or can it be computed?

also at the moment this model servers the purpose of capturing a DCS module which can be a map or an aircraft. We might want to think about breaking this apart using an Abstract model and putting the aircraft specific shiz on a dependent model.

Reply:
initially I had a method that would map it, but if we add or change airframes we will have to commit changes to the code. this way we can make a quick admin change.
As far as I understand it cannot be computed on the fly.

Reply:
Sorry, now i remember that this is the livery directory.

Reply:
i have to verify this later when i'm near my pc but i'd hazzard a guess that the directory name is the same as the name property of the DCS Unit object. 

when we merge in the stats tracking i might update this model so that name corresponds to the DCS naming convention and creating a display name for the website so that we get DCS to do our work for us whenever there is a new module we capture this info automatically. 

that if it all works and i've learned that i should be suspicious.

Reply:
would def be nice if we can link those up together. Let me know